.gitignore

```diff
@@ -6,3 +6,4 @@
 /local.properties
 $catalog.json
 /stagedRepositoryId
+/*/build
```

Comment:
I was thinking if:

```
!*/src*/**/build/
!*/test*/**/build/
```

was going to be required, as I added `build` once here and some files in `src/something/build` were missing. But the `/*/build` would only match a build folder in a root subfolder, right?